### PR TITLE
Link Visit to Nurse Views

### PR DESCRIPTION
**Pull request by carodr50**

### CODE REVIEW

File: src/application/healthSoftware/views/VisitExamQuestions.java [hunk 1]
```diff
@@ -42,12 +46,30 @@ public Region getLayout() {
 		content.setPrefHeight(500);
 		content.setSpacing(15);
 		layout.getChildren().add(content);
+		
+		HBox allergiesInput = makeCenteredInputElement("Allergies");
+		TextField allergiesField = (TextField) allergiesInput.getChildren().get(0);
```
Comment (Brayzure) [inline src/application/healthSoftware/views/VisitExamQuestions.java]: Got an error, pretty sure these should be `TextArea`, not `TextField`.

File: src/application/healthSoftware/views/VisitExamQuestions.java [hunk 2]
```diff
@@ -42,12 +46,30 @@ public Region getLayout() {
...
+		
+		allergiesField.textProperty().addListener((observable, oldValue, newValue) -> {
+			allergies = newValue;
+		});
+		
+		HBox healthConcernsInput = makeCenteredInputElement("Health Concerns");
+		TextField healthConcernsField = (TextField) healthConcernsInput.getChildren().get(0);
```
Comment (Brayzure) [inline src/application/healthSoftware/views/VisitExamQuestions.java]: Here too.

**Review by Brayzure — CHANGES_REQUESTED** [review]
Left a couple comments, otherwise looks good.

**Comment by carodr50** [comment]
Those should be fixed now

**Pull request merged**